addons/project_mail_plugin/controllers/mail_plugin.py

```diff
@@ -20,6 +20,10 @@ def _get_contact_data(self, partner):
         supports it.
         """
         contact_values = super(MailPluginController, self)._get_contact_data(partner)
+
+        if not request.env['project.task'].check_access_rights('create', raise_exception=False):
```

Comment:
Can we maybe add a small comment or docstring as to why we check the "create" right and not the "read" one?
I think that's because this allows creating projects on the plugin's side as well but it's not super clear from reading the code.

(Same for the leads then, and for tickets on the enterprise PR).